.github/ISSUE_TEMPLATE/improvement.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Improvement
```

Comment:
❓ Are "New Feature" and "Improvement" the same thing? Should they have different templates?

Reply:
I'm not sure if the templates would differ aside from auto assigning tags and maybe some of the comments within the description. How do you think the templates would differ?

Reply:
I'm not sure that they would I was posing it more so as a question. This works for now 👍

Reply:
They're definitely different but I'm not sure (yet) whether they have different content needs or that we would automate labeling and/or assigning them much differently.